### PR TITLE
Mlt

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is linux only and ready for review!

File: recipes/mlt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - lame
+    - alsa-lib
+    - pulseaudio
+    - ladspa
+    - jack
+    - fftw
+  run:
+    - libxml2
+    - jack
+    - fftw
```
Comment (beckermr) [inline recipes/mlt/meta.yaml]: fftw has a run export now so you can remove this.

Comment (scopatz) [reply recipes/mlt/meta.yaml]: Ok

**Comment by scopatz** [comment]
Ready again!

**Pull request merged**